src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -1893,6 +1893,7 @@
         "REPO_NAME": "Repository Name",
         "TOOLTIP": "All filters except CVSS3 only support exact matches",
         "NO_VUL": "We could not find any vulnerability",
-        "INVALID_VALUE": "Invalid range"
+        "INVALID_VALUE": "Invalid range",
+        "PAGE_TITLE_TOOLTIP": "The total artifact count includes artifact accessories, child artifacts of the image index, and child artifacts of CNAB artifacts"
```

Comment:
The total artifact count also includes OCI artifact accessories, OCI Image Index child artifacts, and CNAB child artifacts